docs/docs/api-reference/core/useResetRecoilState.md

```diff
@@ -5,6 +5,8 @@ sidebar_label: useResetRecoilState()
 
 Returns a function that will reset the value of the given state to its default value.
 
+Using **useResetRecoilState()** allows a component to set the default value without re-rendering when the value changes.
```

Comment:
"allows a component to reset the state to its default value without subscribing to re-render whenever the state changes."

Reply:
Done. tThe description was updated based on proposed text.